nevergrad/optimization/base.py

```diff
@@ -164,6 +170,22 @@ def num_tell_not_asked(self) -> int:
         """
         return self._num_tell_not_asked
 
+    def set_constraints_management(self, max_constraints_trials: int, constraint_penalization: float, constraint_penalty_exponent: float,
```

Comment:
add defaults parameters to it, so that we can modify them one by one
Or better yet, default values in the init, and defaults to None in this function (and only non-None inputs will update the value)

Reply:
Done, thx.